toolbox/message_tools/bad_message_wizard.py

```diff
@@ -184,6 +185,29 @@ def reset_bad_message_cache():
         print('')
 
 
+def reset_resolved_bad_messages():
+    confirmation = input(f'Confirm you want to reset '
+                         f'resolved bad messages by responding "{colored("yes", "cyan")}": ')
+    if confirmation == 'yes':
+        option = input('Resolve messages not seen in in the last? (default 300s): ')
```

Comment:
A little rewording suggestion, since this doubles as the documentation
```suggestion
        option = input('Clear message exceptions unseen for how many seconds? Press ENTER for default 300: ')
```